translations/base-fr.yaml

```diff
@@ -794,6 +869,21 @@ settings:
             description: >-
                 Le jeu est divisé en secteurs de 16×16 cases. Si ce réglage est activé, les limites de chaque secteur sont affichées.
 
+        pickMinerOnPatch:
+            title: Choisir un extracteur sur un emplacement de ressource.
+            description: >-
+                Activé par défaut, sélectionne l'extracteur si vous utilisez la pipette lors du survol d'un emplacement de ressource.
+
+        simplifiedBelts:
+            title: Convoyeurs simplifiés (Laid)
```

Comment:
`moche` is totally a better translation than `laid`

Reply:
ok!